xbmc/interfaces/Builtins.cpp

```diff
@@ -1851,6 +1852,10 @@ int CBuiltins::Execute(const std::string& execString)
   {
     g_application.StopPVRManager();
   }
+  else if (execute == "pvr.searchmissingchannelicons")
+  {
+    g_application.SearchMissingChannelIcons();
```

Comment:
yep, because the check of `IsStarted()` is done in `CPVRManager::SearchMissingChannelIcons`